src/sass/pages/_home.scss

```diff
@@ -0,0 +1,74 @@
+.hero-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```

Comment:
You will often use display: flex; justify-content: center; align-items: center;  create one mixin for that and use where needed